docs/configuration.asciidoc

```diff
@@ -85,6 +87,41 @@ In this case, you can set the log level of the agent with `ElasticApm:LogLevel`,
 }
 ----
 
+[[configuration-on-asp-net]]
+=== Configuration on ASP.NET
+
+When monitoring ASP.NET applications the agent uses two source of configuration:
+Web.config and environment variables.
+Web.config has precedence over environment variables which means that
+the agent first tries to find a configuration option value by its key in Web.config.
+Only if it's not present then the agent tries to look for it among environment variables
+and if it's not present there as well the agent falls back on the option's default value.
+
+You can find the key of each configuration option
+in the `IConfiguration or Web.config key` column of the corresponding option's description.
+
+[float]
+[[configuration-on-asp-net-sample-config]]
```

Comment:
Same story with this ID. There's no need to mention `configuration` and `config` in the ID. This really should be shortened.

Reply:
I see - please see my comment above.